group/lc/main.tf

```diff
@@ -3,13 +3,13 @@
 ## Creates security group
 resource "aws_security_group" "sg_asg" {
   description = "${var.stack_item_fullname} security group"
-  name_prefix = "${var.stack_item_label}-asg-"
+  name_prefix = "${var.stack_item_label}${var.sg_name_suffix}"
```

Comment:
In this case, we need a little extra trickery. Our default would be to use the `name_prefix`. We should actually probably do this with the asg as well. If someone does set the override, we should use `name`.